index.sh

```diff
@@ -47,4 +47,4 @@ function buildIndex() {
     echo "]"
 }
 
-buildIndex devfiles
+buildIndex
```

Comment:
To be consistent with `check_mandatory_fields.sh` and other scripts (like those in plugin-registry/scripts/ folder) the path to be queried should be passed in as cmdline var from the Dockerfile invocation.